README.md

```diff
@@ -1 +1,13 @@
 # WSL
+# Text file information reader 
+This program will display the following information about a given text file:
+-File size
+-Number of characters
+-Creation date 
+-Last modification date
```

Comment:
I like this. The requirements you wrote here are called `functional requirements` (feel free to check the definition and other examples).